docs/evaluation/tutorials/eval_testing_tutorial.mdx

```diff
@@ -0,0 +1,813 @@
+import Tabs from "@theme/Tabs";
+import CodeBlock from "@theme/CodeBlock";
+import {
+  CodeTabs,
+  typescript,
+  python,
+} from "@site/src/components/InstructionsWithCode";
+
+# LangSmith Testing Framework tutorial
```

Comment:
Title should include `evals` IMO

`Add evals to your app using Pytest or Vitest/Jest`